app/src/components/Scheduling/CalendarView.js

```diff
@@ -0,0 +1,85 @@
+import { React, useState, useEffect } from 'react';
+import DayPicker, { DateUtils } from 'react-day-picker';
+import styles from './CalendarView.css';
+import Header from './Header';
+import { Button } from '../../store/index';
+import { useStyles } from './CalendarViewStyles';
+
+export default function CalendarView() {
+  const classes = useStyles();
+  const [selectedDays, handleDayChange] = useState([]);
```

Comment:
Curious why you broke the pattern here of `[selectedDays, setSelectedDays]`?

Reply:
Oh that was just because the docs and we had forgotten to change that pattern in the code. Will get that fixed!